content/docs/reference/spec/migration/platform-api-0.4-0.5.md

```diff
@@ -0,0 +1,47 @@
++++
+title="Platform API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide describes changes from platform API 0.4 to 0.5, who these changes are relevant for,
```

Comment:
```suggestion
This guide describes changes from platform API 0.4 to 0.5, who these changes impact,
```

Reply:
Sounds good to me - see also [comment](https://github.com/buildpacks/docs/pull/278#discussion_r552847532) for placement of this sentence.